src/discrete-layer/models/discreteLayersStore.ts

```diff
@@ -45,14 +50,41 @@ export const discreteLayersStore = types
           // eslint-disable-next-line @typescript-eslint/ban-ts-comment
           // @ts-ignore
           const result = yield  Promise.resolve(MOCK_DATA_IMAGERY_LAYERS_ISRAEL);
-          self.layersImages = result.map(item => ({...item, selected:false}));
+          self.layersImages = filterBySearchParams(result).map(item => ({...item, selected:false}));
         } catch (error) {
           console.error(error);
           self.state = ResponseState.ERROR;
         }
       }
     );
 
+    function filterBySearchParams(layers: ILayerImage[]): ILayerImage[] {
```

Comment:
add todo: to remove this method when we will get a real data

Reply:
Done